### ISSUE TITLE
ls command doesn't support Chinese character filename, will return empty result

### ISSUE THREAD

**Issue by Nizifan** [Alluxio/alluxio#14724]
**Alluxio Version:**
2.5.2

**Describe the bug**
when ls on a directory, if the directory contains Chinese character(unicode 16 I suppose), the ls will return empty result which confuse user.

**To Reproduce**
root@jssz-bigdata-datanode-2050:/opt/alluxio-release-2.5.0-2-SNAPSHOT# alluxio fs  mount --option aws.secretKey=aeef4e8cce5a6e8b55548b4788741bd3   --option aws.accessKeyId=57d71c51a2dae1ea --option alluxio.underfs.s3.endpoint=http://jssz-boss.bilibili.co /cache s3://cv_data/chenweidong/
Mounted s3://cv_data/chenweidong at /cache
root@jssz-bigdata-datanode-2050:/opt/alluxio-release-2.5.0-2-SNAPSHOT# alluxio fs ls /cache

root@jssz-bigdata-datanode-2050:/opt/alluxio-release-2.5.0-2-SNAPSHOT# alluxio fs cat /cache/fpmodel_v2.py
import torch
import torch.nn as nn

class FPModel(nn.Module):
    def __init__(self, inp, hp, cls_num):
....
root@jssz-bigdata-datanode-2050:/opt/alluxio-release-2.5.0-2-SNAPSHOT# alluxio fs ls /cache/imdb
          11705       PERSISTED 12-21-2021 07:20:20:891  DIR /cache/imdb/00
          10005       PERSISTED 12-21-2021 07:20:20:892  DIR /cache/imdb/11
          16641       PERSISTED 12-21-2021 07:20:20:892  DIR /cache/imdb/01
          20982       PERSISTED 12-21-2021 07:20:20:892  DIR /cache/imdb/12
          16449       PERSISTED 12-21-2021 07:20:20:892  DIR /cache/imdb/02
          17814       PERSISTED 12-21-2021 07:20:20:892  DIR /cache/imdb/13
          12423       PERSISTED 12-21-2021 07:20:20:892  DIR /cache/imdb/03
...

root@jssz-coeus-alluxio-01:~# aws --endpoint http://jssz-boss.bilibili.co s3 ls s3://cv_data/chenweidong/
.....
2020-12-08 12:20:37     144978 人脸关键点标注数据介绍.pdf

**Comment by ljl1988com** [comment]
I test which are all right with alluxio version 2.5.0-2, 2.6.0 and 2.7.0. Can you give some server logs?

![1648522965(1)](https://user-images.githubusercontent.com/3060835/160524604-ec22801a-5def-4678-8298-50b42507de19.png)

**Comment by Nizifan** [comment]
@ljl1988com 
Yes, using Alluxio fs ls can see the file, but the fuse POSIX list return empty.

I provide example in above description:
'''
root@jssz-bigdata-datanode-2050:/opt/alluxio-release-2.5.0-2-SNAPSHOT# alluxio fs ls /cache

root@jssz-bigdata-datanode-2050:/opt/alluxio-release-2.5.0-2-SNAPSHOT# alluxio fs cat /cache/fpmodel_v2.py
import torch
import torch.nn as nn
'''

**Comment by ljl1988com** [comment]
@Nizifan I test fuse mount with newest version. Command `ls` in the mount path is right.

**Comment by secfree** [comment]
Maybe I can share some experience about this issue.

We have several alluxio clusters and there are users accessing it through alluxio-fuse.
Our users encountered the exception `InvalidPathException: Malformed input or input contains unmappable characters` several times when there are files with Chinese character names. 
This exception may be thrown from the AlluxioMaster, AlluxioWorker and AlluxioFuse processes.

I made some investigation and found that `LANG=en_US.UTF-8` needs to be set in the environment of the shell while starting these process. I added `export LANG=en_US.UTF-8` to `conf/alluxio-env.sh` and it resolved this issue.

One more thing to take care is that the environment variable `LANG` may be override by `locale` settings like `LC_ALL`, `LC_NAME`. So it is better to set `locale` related variables to `en_US.UTF-8` too.

Hope this helps.
